x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/settings/custom_links.spec.ts

```diff
@@ -6,10 +6,41 @@
  */
 
 const basePath = '/app/apm/settings/custom-links';
+const deleteCustomLink = () => {
+  // delete customLink if exists
+  const kibanaUrl = Cypress.env('KIBANA_URL');
+  cy.request({
+    log: false,
+    method: 'GET',
+    url: `${kibanaUrl}/internal/apm/settings/custom_links`,
+    body: {},
+    headers: {
+      'kbn-xsrf': 'e2e_test',
+    },
+    auth: { user: 'editor', pass: 'changeme' },
+  }).then((response) => {
+    const customLinkId =
+      response.body.customLinks.length > 0 && response.body.customLinks[0].id;
+    if (customLinkId) {
```

Comment:
Probably better to iterate over all of them if there are multiple instead of only deleting the first one